api/base/urls.py

```diff
@@ -42,6 +42,7 @@
                 url(r'^logs/', include('api.logs.urls', namespace='logs')),
                 url(r'^metaschemas/', include('api.metaschemas.urls', namespace='metaschemas')),
                 url(r'^schemas/', include('api.schemas.urls', namespace='schemas')),
+                url(r'^metrics/', include('api.metrics.urls', namespace='metrics')),
```

Comment:
This is probably better off in the `/v2/_/` namespace, since it's an unsupported, private endpoint.